### PR TITLE
Upgrade scicloj ml deps

### PR DESCRIPTION
**Pull request by behrica**

### CODE REVIEW

**Comment by daslu** [comment]
Nice, I'll look a little later.

**Comment by behrica** [comment]
I added an auto generated list of all direct libs with version and licence.

File: deps.edn [hunk 1]
```diff
@@ -7,12 +7,16 @@
         generateme/fastmath                       {:mvn/version "3.0.0-alpha3"}
         aerial.hanami/aerial.hanami               {:mvn/version "0.20.1"}
         org.scicloj/tableplot                     {:mvn/version "1-alpha14.1"}
-        org.scicloj/metamorph.ml                  {:mvn/version "0.10.3"}
         org.scicloj/scicloj.ml.smile              {:mvn/version "7.4.3"}
+        org.scicloj/hanamicloth                   {:mvn/version "1-alpha8"}
```
Comment (daslu) [inline deps.edn:11]: This was replaced by Tableplot

**Review by daslu — CHANGES_REQUESTED** [review]
@behrica that looks great.

My only comment is that Hanamicloth was replaced by Tableplot so it is not needed.

File: deps.edn (hunk 1 above)
Comment (behrica) [reply deps.edn]: solved and rebased

**Comment by daslu** [comment]
Great, merging

**Pull request merged**